frameworks/Java/beyondj/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+
+rm beyondj-launcher/src/main/resources/launchers/webapp-launchers/beyondj-service.war
+mvn clean install
+cp beyondj-service/target/beyondj-service.war beyondj-launcher/src/main/resources/launchers/webapp-launchers/
+mvn clean install
+rm -rf beyondj-launcher/deploy/
+mkdir beyondj-launcher/deploy/
+
+cp beyondj-launcher/target/beyondj.jar beyondj-launcher/deploy/
+cd beyondj-launcher/deploy/
+jar xvf beyondj.jar
+rm beyondj.jar
+jar -cvf0M beyondj.jar META-INF *
+sudo cp beyondj.jar /var/www/html/beyondj.com/
```

Comment:
I don't like this line - it should not require sudo and it should never install anything outside of `$IROOT`